packages/twenty-website/src/content/developers/self-hosting/setup.mdx

```diff
@@ -139,84 +139,86 @@ yarn command:prod cron:calendar:ongoing-stale
 yarn command:prod cron:workflow:automated-cron-trigger
 ```
 
-# Setup Environment Variables
+# Configuration Management
 
-## Frontend
+Twenty uses a hybrid approach for configuration management, combining environment variables with an admin panel interface.
 
-<ArticleTable options={[
-    ['REACT_APP_SERVER_BASE_URL', 'http://localhost:3000', 'Url of backend server'],
-    ['GENERATE_SOURCEMAP', 'false', 'Generate source maps for debugging'],
-    ['CHROMATIC_PROJECT_TOKEN', '', 'Chromatic token used for CI'],
-    ]}></ArticleTable>
+## Admin Panel Configuration
 
+**Most configuration can be managed through the Admin Panel** once your instance is running:
```

Comment:
Can we have this run automatically so that users don't have to think about it during setup?

 "yarn command:prod cron:messaging:messages-import
yarn command:prod cron:messaging:message-list-fetch
yarn command:prod cron:calendar:calendar-event-list-fetch
yarn command:prod cron:calendar:calendar-events-import
yarn command:prod cron:messaging:ongoing-stale
yarn command:prod cron:calendar:ongoing-stale
yarn command:prod cron:workflow:automated-cron-trigger" 

Maybe create one super command or integrate it as part of the upgrade... Never dug into this so not 100% sure what they do. But it would help us keep the docs focused on Env Var only

Reply:
Can be in another PR of course